src/main/java/com/ita/if103java/ims/controller/WarehouseController.java

```diff
@@ -1,31 +1,61 @@
 package com.ita.if103java.ims.controller;
 
+import com.ita.if103java.ims.dto.WarehouseDto;
 import com.ita.if103java.ims.dto.WarehouseDto;
 import com.ita.if103java.ims.mapper.WarehouseDtoMapper;
 import com.ita.if103java.ims.service.WarehouseService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please fix your IDEA configs to not replace imports by * up to 999 same classes from the package
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
already fix it